src/cli/cli.hpp

```diff
@@ -866,6 +867,9 @@ class Interpreter : public Output
 #endif
 #endif // OPENTHREAD_FTD || OPENTHREAD_MTD
         {"version", &Interpreter::ProcessVersion},
+#if OPENTHREAD_FTD && OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE
+        {"vrcheckskip", &Interpreter::ProcessEnableVrCheck},
```

Comment:
```suggestion
        {"tvcheckskip", &Interpreter::ProcessThreadVersionCheckSkip},
```

`vrcheckskip` means skipping (disabling) VR check
`EnableVrCheck` means enabling VR check. 

Aren't they opposite?

Reply:
fixed.